### PR TITLE
Docs: using in clearcalcs page updated for custom diagram interactivity

### PR DESCRIPTION
**Pull request by michaellisitsa**
- [Modify] Docs: Update screenshots of builder so they all have consistent widgets with the boilerplate example 
- [Add] Docs: Add steps for updating default values in preset editor including clicking on diagram to save storedParams

### CODE REVIEW

**Review by JayserBeam — CHANGES_REQUESTED** [review]
Otherwise I can follow along quite easily.
Just need to know if that picture is redundant or hasn't been placed

File: docs/using-in-clearcalcs.md [hunk 1]
```diff
@@ -25,35 +25,48 @@ The builder is a web-based calculator creator. All ClearCalcs calculators are ad
...
 
-![Screenshot of the builder](_media/using-in-clearcalcs/builder-main-page.png ":size=400")
+![Screenshot of the builder](_media/using-in-clearcalcs/builder-main-page.png ":size=600")
 
 </div>
 
-3. Upload `output/index.html` (interactive diagram) and/or `output/compiled.js` (static diagram) into the ClearCalcs builder.
-4. Add values or references to other sheet widgets against each parameter
+3. Upload `output/index.html` (interactive diagram) and/or `output/compiled.js` (static diagram) into the ClearCalcs builder. The default example is shown with interactive circle border [src/static/interface.ts](https://github.com/ClearCalcs/custom-diagram-boilerplate/blob/main/src/static/interface.ts) included.
+4. Add values or references to other sheet widgets against each param. The storedParams keys will also be listed.
```
[the remark this answers is not in the document]
Comment (michaellisitsa) [reply docs/using-in-clearcalcs.md:37]: Very true. This image was supposed to be the last step in the Building instructions. Now added 5dea5cc47202c406dd7aaf30fef42c5991390ce0

**Comment by michaellisitsa** [comment]
@JayserBeam Ready for re-review

**Review by JayserBeam — APPROVED** [review]
Looks good. Send it

**Pull request merged**